app/controllers/editions_controller.rb

```diff
@@ -118,4 +153,8 @@ def redirect_url
   def description(resource)
     resource.format.underscore.humanize
   end
+
+  def permitted_params
+    params.permit(edition: %i[title overview in_beta body major_change change_note])
```

Comment:
Not sure if this matters but I have just been learning about 'strong parameters' and I have only seen this written as:
`params.require(:edition).permit(%i[title overview in_beta body major_change change_note])`
This will raise an error if the :edition attribute is missing. Maybe that isn't what you want here though?

[Use Strong Parameters](https://www.mintbit.com/blog/securing-ruby-on-rails-applications-part-3-use-strong-parameters)

Reply:
My feeling is that that makes a lot of sense and helps with the naming above. So I'm changing this accordingly. 👍